anomaly-detection/build.gradle

```diff
@@ -0,0 +1,109 @@
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.4'
+}
+
+apply plugin: "java"
+apply plugin: "distribution"
+apply plugin: 'application'
+
+sourceCompatibility = "1.8"
+mainClassName = 'io.pravega.anomalydetection.ApplicationMain'
+applicationDefaultJvmArgs = ["-Dlogging.config=file:conf/logback.xml"]
+
+archivesBaseName = 'pravega-flink-anomaly-detection'
+
+dependencies {
+    compileOnly "org.slf4j:slf4j-api:1.7.14"
+    compileOnly "ch.qos.logback:logback-classic:1.1.7"
```

Comment:
It makes no sense to me to have logback as  'compileOnly' dependency.   It is usually a 'runtime' dependency because code never directly uses its API.